app/views/LivechatEditView.js

```diff
@@ -37,10 +43,11 @@ Title.propTypes = {
 };
 
 const LivechatEditView = ({
-	user, navigation, route, theme
+	user, navigation, route, theme, editOmnichannelContact, editLivechatRoomCustomfields
```

Comment:
Can you add a margin between the Save button and the tags input field so it doesn't look like this?
![image](https://user-images.githubusercontent.com/36211892/128925042-8d828338-696e-4b0e-b169-a04a1d2dde29.png)